README.md

```diff
@@ -120,7 +120,13 @@ The `any` field is a list and holds one or more `resources` objects, which conta
   - `matchLabels`: a map of key-value pairs.
   - `matchExpressions`: a list of label selector requirements evaluated as a logical AND operation. **(not implemented yet)**
 
-- `containerSelector`: selects the container(s) in the matched pods or deployments where the trap is deployed. It supports the same pattern syntax as [`regexp`](https://github.com/google/re2/wiki/Syntax). The default value is empty, and the trap is deployed in all containers in the matched pods.
+- `containerSelector`: selects the container(s) in the matched pods or deployments where the trap is deployed.
+  - if this field is prepended by `regex:`, the rest of the string will represent a regular expression matched with go [regexp](https://golang.org/s/re2syntax) library.
+  - if the field is prepended by "glob:", then this is a filesystem-style regex, as described in go [filepath.Match](https://pkg.go.dev/path/filepath#Match) library
```

Comment:
[FORMATTING, CLARITY] I think the proper name is "globbing" (https://en.wikipedia.org/wiki/Glob_(programming))
```suggestion
  - if the field is prepended by `glob:`, then this is a glob pattern, as described in go [filepath.Match](https://pkg.go.dev/path/filepath#Match) library
```